inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/service/RecommendationServiceImpl.java

```diff
@@ -186,6 +187,24 @@ public Recommender getRecommender(long aId)
         return entityManager.find(Recommender.class, aId);
     }
 
+    @Override
+    @Transactional
+    public List<Recommender> getRecommenderIfActive(Project aProject, String aLayer, String aTool)
```

Comment:
We have "active" and "enabled" recommenders. A recommender is "active" if it is used to generate suggestions (i.e. if its evaluation score surpasses the threshold or if it is "always active"). Thus better rename to `getEnabledRecommenders()` to avoid confusion.

Reply:
Updated